### ISSUE TITLE
the problem about eval.py 

### ISSUE THREAD

**Issue by LearningJack** [odegeasslbc/FastGAN-pytorch#34]
In the eval.py code, why you annotated the net.ig.eval() ? So you didn't use net_ig.eval(). But when we do the test, we would load the model and then do the model.eval().

**Comment by odegeasslbc** [comment]
feel free to uncomment it and check out the result difference. Its a little bit involved about why I comment out .eval(). Basically it is because the running mean and running std in the BatchNorm layer does not saved properly when I do EMA optimizing of the Generator. It common in many otherprojects where we don't use eval mode on models trained as GAN.

**Comment by yangyu615** [comment]
What's the difference of optimizerG.state_dict() and netG.state_dict(), thank you！
